packages/core-data/src/actions.js

```diff
@@ -643,7 +643,17 @@ export function* __experimentalBatch( requests ) {
 			);
 		},
 	};
-	const resultPromises = requests.map( ( request ) => request( api ) );
+	const resultPromises = [];
+	const awaitNextFrame = () =>
+		__unstableAwaitPromise( new Promise( window.requestAnimationFrame ) );
```

Comment:
Even though `requestAnimationFrame` only accepts a single parameter, it's still an anti-pattern to directly pass it as the promise callback. See [this article](https://jakearchibald.com/2021/function-callback-risks/) for more info.

Reply:
I can't argue with that – great catch and thank you for the link @kevin940726!